proposal/package-importer.d.ts.md

```diff
@@ -216,19 +216,28 @@ using symlinks if this behavior is desired.
 
 [ECMAScript modules]: https://nodejs.org/api/esm.html#no-node_path
 
+The [Node resolution algorithm] requires a `parentURL`, used for determining
+where in the file system to start searching for a module if a `pkg:` URL is
+being resolved in a source somewhere other than a file on disk. For instance,
+when compiling a string like `compileString('@use "pkg:bootstrap";')`, we don't
+know where to start looking for the Bootstrap module. We considered
+`require.main.filename` and the current working directory, but found that
+neither would allow for all use cases. We decided to allow users to specify an
+entry point path, defaulting to `require.main.filename`.
+
 ## Types
 
 ```ts
 import {FileImporter, Importer} from '../spec/js-api/importer';
 ```
 
-### `nodePackageImporter`
+### `NodePackageImporter`
 
 ```ts
-export type NodePackageImporter = {
-  _NodePackageImporterBrand: unknown;
-};
-export declare const nodePackageImporter: NodePackageImporter;
+export class NodePackageImporter {
+  entryPointPath?: string;
```

Comment:
I don't think we need to expose this publicly once the class is constructed. (And if we did, we wouldn't want it to be `undefined`, since it does have a well-defined default value.)

Reply:
Fixed in 64dc0a12696821062abfae2c1d33c9247b1fe1b6